src/User/src/Entity/User.php

```diff
@@ -103,18 +100,6 @@ public function setStatus(UserStatusEnum $status): self
         return $this;
     }
 
-    public function isDeleted(): bool
```

Comment:
Let's keep this as a helper method, so we do't have to repeat comparing user->status against `UserStatusEnum::Deleted`. Move the metod after `isActive()` so we have all helper methods in one place.